src/plugins/expression_shape/public/components/index.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { lazy } from 'react';
+
+export * from './shape_component';
```

Comment:
OK, so it looks like exporting * from here exports the component itself, which causes it to end up in the main bundle, defeating the purpose of the lazy wherever that is.  

Or just export it as lazy from here to keep it out of the bundle and import the lazy where it's currently used.

Reply:
Done.